app/views/components/super-navigation-header/_header-content.html.erb

```diff
@@ -0,0 +1,21 @@
+<div class="govuk-header__content govuk-header__super-navigation-content">
+  <button type="button" class="govuk-header__menu-button govuk-js-header-toggle govuk-header__super-navigation-menu-button" aria-controls="navigation" aria-label="Show or hide navigation menu">
+    Menu
+  </button>
+  <nav>
+    <ul id="navigation" class="govuk-header__navigation govuk-header__super-navigation-items" aria-label="Navigation menu">
+      <% navigation_links.each do | link | %>
+        <li class="govuk-header__navigation-item govuk-header__super-navigation-item">
+          <a class="govuk-header__link govuk-header__super-navigation-item-link" href="<%= link["href"] %>">
+            <%= link["label"] %>
+          </a>
+        </li>
+      <% end %>
+      <li class="govuk-header__navigation-item govuk-header__super-navigation-item govuk-header__super-navigation-item--search">
+          <a class="govuk-header__link govuk-header__super-navigation-item-link govuk-header__super-navigation-item-link--search" href="/search">
+            <span class="govuk-header__super-navigation-item-link-text--search">Search</span>
+          </a>
+        </li>
```

Comment:
There's some space characters between each list item that's causing each link to have a slither of space between each of them - and frustatingly using `<%-` doesn't seem to be working here.

I mention it because I think that this looks to be on purpose, but the CSS says otherwise; and the space between "Government activites" and "Search" looks as if it's broken because of the gap on the bottom right corner:

![image](https://user-images.githubusercontent.com/1732331/120812767-32c33000-c545-11eb-8297-e6a693e309e3.png)

Reply:
Fixed